### PR TITLE
feat: Modal component

### PR DESCRIPTION
**Pull request by ParthParolekar**
Added Modal component

### CODE REVIEW

**Review by abhisheksahani-nova — APPROVED** [review]
Looks good

File: Components/modal.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+.modal-title {
+  margin-bottom: 1rem;
+}
+
+.modal-dismiss {
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
```
Comment (VLeads) [inline Components/modal.css:18]: Also add cursor pointer to dismiss.

**Review by VLeads — COMMENTED** [review]
Also in card component. There is no border radius on all corner. Have you did it intentionally or maybe you missed it.

File: Components/modal.css (hunk 1 above)
Comment (ParthParolekar) [reply Components/modal.css:18]: Will do it.

**Pull request merged**